### PR TITLE
FIX: fix seed issue in ICA tutorial

### PR DESCRIPTION
**Pull request by dengemann**
and another fix, seed of the evil.

### CODE REVIEW

**Comment by dengemann** [comment]
cc @agramfort @Eric89GXL @jaeilepp

File: tutorials/plot_artifacts_correction_ica.py [hunk 1]
```diff
@@ -136,52 +140,58 @@
 # semi-automatic. Corrmap hence takes at least a list of ICA solutions and a
 # template, that can be an index or an array. As we don't have different
 # subjects or runs available today, here we will fit ICA models to different
-# parts of the recording and then use a user-defined template from the first
-# part for detecting corresponding components in the other parts. The following
-# block of code in addresses this point and should not be copied, ok?
-#
+# parts of the recording and then use as a user-defined template the ICA
+# that we just fitted for detecting corresponding components in the three "new"
+# ICAs. The following block of code in addresses this point and should not be
```
Comment (agramfort) [inline tutorials/plot_artifacts_correction_ica.py]: in addresses -> addresses

File: tutorials/plot_artifacts_correction_ica.py [hunk 2]
```diff
@@ -136,52 +140,58 @@
...
 # ``icas = [mne.preprocessing.read_ica(fname) for fname in ica_fnames]``
-print(icas)
+print(icas_from_other_data)
+
+###############################################################################
+# use our previous ICA as reference.
+reference_ica = ica  
 
 ###############################################################################
 # Investigate our reference ICA, here we arbitrarily say it's the first.
```
Comment (agramfort) [inline tutorials/plot_artifacts_correction_ica.py]: comment do not match here

File: tutorials/plot_artifacts_correction_ica.py [hunk 3]
```diff
@@ -136,52 +140,58 @@
...
-reference_ica = icas[0]
 reference_ica.plot_components()
 
 ###############################################################################
-# With random seed = 42 we see that IC number 13 from run 1 looks like an EOG.
-reference_ica.plot_sources(eog_average, exclude=[12])
+# Which one is the bad EOG component?
+# Well we cannot say this across computers and operating systems because
+# ICA solutions are not deterministic. Fixing the random seed will only work
+# for sure within the same Python session, so watch out.
```
Comment (agramfort) [inline tutorials/plot_artifacts_correction_ica.py]: this is for us to know. It's too much information for a tutorial. This comment is a bit irrelevant here

**Comment by dengemann** [comment]
addressed comments.

**Comment by agramfort** [comment]
fix pep8 to make travis happy and merge

**Comment by agramfort** [comment]
merged by rebase after pep8 fix

thx @dengemann

**Pull request closed without merging**